explainers/navigation-to-bundles.md

```diff
@@ -0,0 +1,83 @@
+# Explainer: Navigation with Web Bundles<br>(a.k.a. Bundled HTTP Exchanges)
+
+Last updated: Oct 29, 2019
+
+(This explainer was originally written and published as: https://docs.google.com/document/d/1KFmtiE3DHgKfQH5-nKtLiacMrXsoKIXQZ-VIMGHMje0/edit?pli=1#)
+
+A [demo video](https://www.youtube.com/watch?v=rs-3R0ji6dA) that tries to show one of the use-case scenarios of Web Bundles (a.k.a. Bundled HTTP Exchanges):
+
+<a href="https://www.youtube.com/watch?v=rs-3R0ji6dA">
+<img src="https://img.youtube.com/vi/rs-3R0ji6dA/0.jpg">
+</a>
+
+Today, loading a website basically means retrieving multiple resources from one or multiple web servers. This gives web a great strength of linkable, indexable, composable and ephemeral, but it is also making it difficult for a website to:
+
+* be shared and loaded in a self-contained way, similar to what packaged apps and other portable data representation formats provide,
+* be composed as a distributable, installable application, or
+* load reliably and quickly even when a site consists of a large number of subresources, and even when the internet access is limited.
+
+Instead, imagine if we could bundle up a “full” website in a single resource file, so that the website could be shared via a SD card or over some p2p protocol, or could be retrieved from a fast cache or a nearby proxy. It would open up various interesting use cases.
+
+**Web Bundle** is a proposal that tries to achieve it. It’s a format that can represent a collection of HTTP resources, and therefore can represent one or multiple web pages (which typically consist of multiple resources, e.g. HTML, javascripts, images and styles) in a single file.  It’s a part of [Web Packaging](https://github.com/WICG/webpackage) proposal and is also known as "**Bundled HTTP Exchanges** ([spec proposal](https://wicg.github.io/webpackage/draft-yasskin-wpack-bundled-exchanges.html))."
+
+This document explains how a Web Bundle can be a navigation target, so that a user can open a given Web Bundle in browsers (UAs) and navigate into the website represented by the Bundle.
+
+## Example Scenarios: Save and browse an unsigned content
+UA can provide **‘Save as bundle’** feature that will dynamically generate an unsigned bundle of the page that a user is currently browsing.  Then anyone can use the feature to get a bundled representation of the current page, which can be browsed later by the same user, or can be shared with a nearby friend via some person-to-person sharing mechanism.
+
+When someone else browses the unsigned bundle, they see some notation in their URL bar that isn’t the URL of the original URL, but they can still browse around and see the JS executing.  Note that in this case the page coming from the unsigned bundle is not given any access to the cookie/storage of the original site.
```

Comment:
@estark37 commented that showing the original URL is probably a bad idea. It sounds like we'll show the package's URL and/or a chip describing what's happening.

```suggestion
When someone else browses the unsigned bundle, they can browse around and see the site, including the results of its JavaScript executing.  Note that in this case the page inside the unsigned bundle is not given any access to the cookies or storage of the original site.
```